### PR TITLE
LOC 7409

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New place, Taqot.

### CODE REVIEW

File: new/LOC7409Taqot.xml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="ti">Tigrinya</language>
```
Comment (thea-m) [inline new/LOC7409Taqot.xml:37]: here ti and below in the actual name gez (also in the other file)

**Review by thea-m — CHANGES_REQUESTED** [review]

File: new/LOC7409Taqot.xml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="DN" when="2024-01-03">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="qushat">
+                    <placeName xml:id="n1" xml:lang="gez">ጣቖት፡</placeName>
```
Comment (eu-genia) [inline new/LOC7409Taqot.xml:48]: ti?

File: new/LOC7411HaddesHeywat.xml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="DN" when="2024-01-09">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="tabiya">                  
+                    <placeName xml:id="n1" xml:lang="gez">ሓድሽ፡ ህይወት፡</placeName>
```
Comment (eu-genia) [inline new/LOC7411HaddesHeywat.xml:48]: ti?

File: new/LOC7409Taqot.xml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+                    <placeName corresp="#n1" xml:lang="en">Ṭāq̲ot</placeName>   
+                </place>              
+            </listPlace><!---->
```
Comment (eu-genia) [inline new/LOC7409Taqot.xml:51]: add at least Tigray as region and Ethiopia as country

File: new/LOC7411HaddesHeywat.xml (hunk 3 above)
Comment (eu-genia) [reply new/LOC7411HaddesHeywat.xml:48]: add at least Tigray as region and Ethiopia as country

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Review by eu-genia — APPROVED** [review]

**Pull request merged**